code/game/objects/items/weapons/cash.dm

```diff
@@ -126,6 +126,9 @@ var/global/list/moneytypes = list(
 			update_icon()
 			to_chat(user, "<span class='notice'>You add [collected] [src.name][amount > 1 ? "s":""] to your stack of cash.</span>")
 
+/obj/item/weapon/spacecash/proc/get_total()//I can't believe this didn't exist here already.
```

Comment:
actually it doesn't, never mind.
could you change the proc at the end of the file though?

```
 		. += C.amount * C.worth
```
becomes
```
 		. += C.get_total()
```